src/routes/common/interceptors/route-logger.interceptor.ts

```diff
@@ -27,6 +27,10 @@ import { Request, Response } from 'express';
  */
 @Injectable()
 export class RouteLoggerInterceptor implements NestInterceptor {
+  private readonly PRE_EXECUTION_LOGGING_ROUTE = '/v1/hooks/events';
+  private readonly PRE_EXECUTION_LOGGING_DETAIL = 'pre-execution-log';
+  private readonly PRE_EXECUTION_LOGGING_STATUS = 202;
```

Comment:
These values are hardcoded, as this can be seen more like a transient feature. 

`202 - Accepted` is used as `formatRouteLogMessage` has a mandatory `statusCode` argument, and I'm trying to not do many changes across classes for this.

Reply:
> Have you considered tackling this in CacheHooksController?

Yes, but I added the logging to this interceptor to be as close as possible to the request cycle start (this interceptor is executed before the code in the controller function body if I'm not wrong).

> Also you don't need to call formatRouteLogMessage unless you want to log all the parameters tied to the route. Is that the case? 🤔

I agree, but I thought it would be nice to reuse the mappings provided by that utility and have the full information about the request in the logs.

Anyway, I don't have a strong opinion regarding these topics, we can move the logging to the controller if you prefer, but as I said if a request execution gets stuck in the interceptor for some reason, the log won't be generated 🙂

Reply:
So we have logs for when the request finishes. That's the expectation and that's why we have the status code available.

With this change we are mixing pre-call logs with post-call logs. This can twist the metrics that we have for `/v1/hooks/events`.

If there's any reason to believe that this interceptor is somehow blocking calls then I can understand the change. Else I do think that `CacheHooksController` or another interceptor specialised in incoming requests is a better fit.